### DISCUSSION TITLE
tutorials/growing-graphql-api-dgs-lab/04-update-models

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#119] [course feedback]
# tutorials/growing-graphql-api-dgs-lab/04-update-models

https://www.apollographql.com/tutorials/growing-graphql-api-dgs-lab/04-update-models

**Comment by mkatsman** [comment]
IMHO, it is easier to just maplist of artists than extracting the id of the first one. I just do not see the use case for it.

**Reply by lizhennessy** [reply]
That's a great point, @mkatsman , and it definitely matches how the Spotify API has implemented their artist information - with potentially _many_ artists assigned to one track. Please feel free to augment your GraphQL schema so that each `Track` has an `artists` field that returns a list of `Artist` types instead, if you'd like! We wanted to keep the example simple with one Artist to one Track, but I certainly see your point too. :)
